### ISSUE TITLE
Information should explicitly include software 

### ISSUE THREAD

**Issue by benbalter** [ombegov/a130#54]
> 'Information' means any communication or representation of knowledge such as facts, data, or opinions in any medium or form, including textual, numerical, graphic, cartographic, narrative, electronic, or audiovisual forms.

"Information" should explicitly include "computer software" (and associated documentation) within the definition. This would mirror the FAR's treatment of "technical data" and would elevate the treatment of software within government to be treated with the same respect as more traditional data. 

As government increasingly relies on software and algorithms to deliver citizen services, software transparency will be essential along side data transparency to allow citizens to check the government's work (and build upon for their own use). If the IRS uses an algorithm to calculate my taxes (or a voting machine to count my vote), and that software is a black box or encumbered by private-sector copyright, government becomes a black box.

**Comment by Neutron** [comment]
I agree with this  - but would clarify that "source code" be used instead of "software."  Source code is information.  Software that is compiled and configured is a product of and processor of Information.  It can be considered information .. but if we don't draw the line somewhere then the problem gets too big in my opinion.

**Comment by thecapacity** [comment]
I can't speak to the "source code" as information vs. product discussion - but I think @benbalter's example about algorithms might be a good way to resolve the distinction(s).

According to google algorithm is "a process or set of rules to be followed in calculations or other problem-solving operations, especially by a computer" - and note it doesn't specify if it's in source or binary form.

So perhaps the language in the definition could suggest
`
"information" means any .... <basically definition as is> _as well as any set of rules or methods conducted in order to derive that information)."
`

Basically trying to suggest (much like open science does) that it's not enough to give just the data, you have to provide the artifacts associated with its creation too.

**Comment by benbalter** [comment]
> but would clarify that "source code" be used instead of "software." Source code is information. 

Great catch. Updated the OP.
